### PR TITLE
revamp contributions section of checkout

### PR DESCRIPTION
**Pull request by raynamharris**
There are a few open issues and PRs related to revamping the wording of the checkout process (https://github.com/carpentries/instructor-training/pull/945, https://github.com/carpentries/instructor-training/issues/1024, https://github.com/carpentries/handbook/issues/489). I tried to incorporate comments from @dhuppenkothen @k8hertweck @ErinBecker @Denubis, but I also moved things a few things around and added subheadings to make the description more cohesive. 

What do y'all think?

### CODE REVIEW

**Comment by Denubis** [comment]
I really like your edits. My main question (I suspect I need more coffee) is: do we ever operationalise "actively maintained?"

File: _extras/checkout.md [hunk 1]
```diff
@@ -49,40 +49,52 @@ and must be requested before the three months are up. To request an extension, p
 ## Part 1: Submit a Small Contribution to One of Our Lessons
 <a name="lesson-change"></a>
 
-Our lessons are maintained and improved by the people who teach them,
-so this part of the checkout procedure gives you a chance to familiarize yourself
+Our lessons and lesson infrastructur are maintained and improved by the people who use them.
```
Comment (k8hertweck) [inline _extras/checkout.md]: infrastructur -> infrastructure

**Comment by k8hertweck** [comment]
One typo correction suggestion, but happy otherwise with how this resolves my issue. 

@Denubis This is one of the topics Lesson Programs are supposed to manage, although we're still figuring out governance for these types of decisions. One operational way of describing "active maintenance" is whether the lesson receives a PR from the infrastructure committee to update the lesson template, so we do track this to a certain extent.

**Comment by Denubis** [comment]
@k8hertweck Ah. My question was more for the language in the lesson. How do we communicate "actively maintained" to our new instructors?

**Comment by k8hertweck** [comment]
The Software Carpentry website has these listed as "Core Lessons," while the Data Carpentry lessons are more compartmentalized into units and have labels like "alpha" for genomics R. Perhaps it might be worthwhile to describe this?

**Comment by dhuppenkothen** [comment]
Love the new rewording! 👍

**Pull request merged**